test/test_other.py

```diff
@@ -2691,35 +2672,49 @@ def test_dwarf_sourcemap_names(self):
       ([], False, False, False),
       (['-g0'], False, False, False),
       (['-g1'], False, False, False),
+      (['-g1', '-O2'], False, False, False),
```

Comment:
Please add a line with plain `-O2` and nothing else (like the old test had) - just to be safe here.

Reply:
Done